test/common/lib/signature-utils.test.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.24 <0.9.0;
+
+import "forge-std/Test.sol";
+import {ECDSA} from "contracts/common/lib/ECDSA.sol";
+import { SignatureUtils } from "contracts/common/lib/SignatureUtils.sol";
+
+contract ExposedSignatureUtils {
```

Comment:
Please consider restricting the funcs mutability to `view` (as the compiler warns)

```bash
Warning (2018): Function state mutability can be restricted to view
 --> test/common/lib/signature-utils.test.sol:9:5:
  |
9 |     function _isValidSignature(
  |     ^ (Relevant source part starts here and spans across multiple lines).

Warning (2018): Function state mutability can be restricted to view
  --> test/common/lib/signature-utils.test.sol:19:5:
   |
19 |     function hasCode(address addr) public returns (bool) {
   |     ^ (Relevant source part starts here and spans across multiple lines).
```

Reply:
Fixed.